File: P1-Linguagem_de_Programação_I/Desafios dos Roteiros - Python/desafio-roteiro06/versao-procedural/ex07.py
nota = int(input("Digite a nota (0-100): "))

if nota >= 90:
    print("Categoria A")
elif nota >= 80:
    print("Categoria B")
elif nota >= 70:
    print("Categoria C")
elif nota >= 60:
    print("Categoria D")
else:
    print("Categoria E")

File: P1-Linguagem_de_Programação_I/Desafios dos Roteiros - C++/desafio-roteiro06.cpp/versao-procedural/ex07.cpp
#include <iostream>
using namespace std;

int main() {

    int nota;

    cout << "Digite a nota: ";
    cin >> nota;

    if (nota >= 90)
        cout << "Categoria A";
    else if (nota >= 80)
        cout << "Categoria B";
    else if (nota >= 70)
        cout << "Categoria C";
    else if (nota >= 60)
        cout << "Categoria D";
    else
        cout << "Categoria E";

    return 0;
}